### PR TITLE
Fix JDF-484, JDF-485: Modify quickstart GAVs, JBoss BOM name for the reo...

### PR DESCRIPTION
**Pull request by sgilda**
...rg

### CODE REVIEW

File: bean-validation/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
     <groupId>org.jboss.quickstarts.eap</groupId>
     <artifactId>jboss-as-bean-validation</artifactId>
-    <version>7.1.2-SNAPSHOT</version>
+    <version>6.2.0-redhat-1</version>
```
Comment (rafabene) [inline bean-validation/pom.xml]: It should be 6.2.0-redhat-SNAPSHOT. Don't you think ?

File: bean-validation/pom.xml [hunk 2]
```diff
@@ -46,12 +46,12 @@
         <version.jboss.maven.plugin>7.4.Final</version.jboss.maven.plugin>
 
         <!-- Define the version of the JBoss BOMs we want to import to specify tested stacks. -->
-        <version.jboss.bom>1.0.7.Final</version.jboss.bom>
+        <version.jboss.bom.eap>1.0.7.Final</version.jboss.bom.eap>
```
Comment (rafabene) [inline bean-validation/pom.xml:49]: EAP version is 6.2.0-redhat-1

File: cdi-add-interceptor-binding/pom.xml [hunk 3]
```diff
@@ -46,12 +46,12 @@
         <version.jboss.maven.plugin>7.4.Final</version.jboss.maven.plugin>
         
         <!-- Define the version of the JBoss BOMs we want to import. The JBoss BOMs specify tested stacks. -->
-        <version.jboss.bom>1.0.7.Final</version.jboss.bom>
+        <version.jboss.bom.wfk>1.0.7.Final</version.jboss.bom.wfk>
```
Comment (rafabene) [inline cdi-add-interceptor-binding/pom.xml:49]: WFK version is 2.4.0-redhat-1

File: bean-validation/pom.xml (hunk 1 above)
Comment (sgilda) [inline bean-validation/pom.xml]: How the heck did that happen? ;-)
Will fix it.

File: cdi-add-interceptor-binding/pom.xml [hunk 4]
```diff
@@ -77,14 +77,14 @@
             <dependency>
                 <groupId>org.jboss.bom</groupId>
```
Comment (rafabene) [inline cdi-add-interceptor-binding/pom.xml:78]: groupId should change to org.jboss.bom.wfk for WFK boms

File: cdi-add-interceptor-binding/pom.xml [hunk 5]
```diff
@@ -77,14 +77,14 @@
...
                 <artifactId>jboss-javaee-6.0-with-tools</artifactId>
-                <version>${version.jboss.bom}</version>
+                <version>${version.jboss.bom.wfk}</version>
```
Comment (rafabene) [inline cdi-add-interceptor-binding/pom.xml:80]: -with-tools is a EAP bom

**Comment by sgilda** [comment]
Good catch. I think it is fixed now.

**Comment by sgilda** [comment]
No, it's not. Hang on. Need to make more changes.

**Comment by sgilda** [comment]
Closing to add more changes.

File: cdi-add-interceptor-binding/pom.xml (hunk 3 above)
Comment (sgilda) [inline cdi-add-interceptor-binding/pom.xml:49]: @rafabene : That gave me a compile error. I believe it should be 2.4.0-redhat-SNAPSHOT. Right?

Comment (rafabene) [inline cdi-add-interceptor-binding/pom.xml:49]: The right GAVs are: org.jboss.bom.eap:jboss-javaee-6.0-with-tools:${version.jboss.bom.eap} AND org.jboss.bom.wfkjboss-javaee-6.0-with-deltaspike:${version.jboss.bom.wfk}

version.jboss.bom.wfk = 2.4.0-redhat-1
version.jboss.bom.eap = 6.2.0-redhat-1

**Pull request closed without merging**